plugins/pluginmanagers/lz-n.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+let
+  lzPluginType =
+    with helpers.nixvimTypes;
+    submodule {
+      options = {
+        name = mkOption {
+          type = str;
+          description = "The plugin's name (not the module name). This is what is passed to the load(name) function.";
+        };
+        enabledInSpec = helpers.mkNullOrStrLuaFnOr bool ''
```

Comment:
I had changed the name to avoid confusion with the `enable` option on the other PR, but here it is not necesary.

```suggestion
        enabled = helpers.defaultNullOpts.mkStrLuaFnOr bool true ''
```
Upstream accepts a lua function that returns a boolean for this option. Wouldn't `mkStrLuaFnOr` be more apropiate?

Reply:
I think it would be better if upstream accepts that

Reply:
Well, `mkBool` will already return a `maybeRaw bool` type, so I don't think it's really necessary to support `strLuaFn` here.

This may just be my personal opinion, but the consistency of always using `__raw` or `helpers.mkRaw` is good for knowing what is lua code and what is a string.